examples/opaque-types.flix

```diff
@@ -3,10 +3,10 @@
 /// would otherwise be the same. For example:
```

Comment:
Should this just say `enum` instead of `opaque type` , or de we need a new explanation?